src/System.Console/src/System/ConsolePal.Windows.cs

```diff
@@ -790,15 +784,6 @@ public static void Clear()
 
         public static void SetCursorPosition(int left, int top)
         {
-            // Note on argument checking - the upper bounds are NOT correct 
-            // here!  But it looks slightly expensive to compute them.  Let
-            // Windows calculate them, then we'll give a nice error message.
-            if (left < 0 || left >= short.MaxValue)
-                throw new ArgumentOutOfRangeException(nameof(left), left, SR.ArgumentOutOfRange_ConsoleBufferBoundaries);
-            if (top < 0 || top >= short.MaxValue)
-                throw new ArgumentOutOfRangeException(nameof(top), top, SR.ArgumentOutOfRange_ConsoleBufferBoundaries);
-            Contract.EndContractBlock();
```

Comment:
This is the same as Console.cs: https://github.com/hughbe/corefx/blob/fbca63abe848380b8d07ebfad84ee23a20f2f17d/src/System.Console/src/System/Console.cs#L328-L332